thinc/backends/numpy_ops.pyx

```diff
@@ -62,19 +62,20 @@ class NumpyOps(Ops):
 
     def asarray(self, data, dtype=None):
         if isinstance(data, self.xp.ndarray):
-            if dtype is not None:
-                return self.xp.asarray(data, dtype=dtype)
-            else:
-                return self.xp.asarray(data)
+            array = self.xp.asarray(data)
```

Comment:
Is this now still needed, since data is already `xp.ndarray`? Also see `Ops.asarray` for the expected semantics.